hasjob/views/helper.py

```diff
@@ -627,6 +627,12 @@ def usessl(url):
     return url
 
 
+@app.template_filter('post_from_md5sum')
```

Comment:
This template filter is a terrible hack. Surely there's a more elegant way to expose this function?